### ISSUE TITLE
Error in creating a new lemma while creating a new sign

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#582]
I created a new dataset, with the first gloss. First created one lemma, all good. Then wanted to change that lemma, and then I get an error that refers to the presumed video, but this video hadn't been uploaded yet!!

Request Method: | POST
-- | --
https://signbank.science.ru.nl/dictionary/lemma/add/36457
1.11
FileNotFoundError
[Errno 2] No such file or directory: '/var/www/signbank/live/writable/glossvideo/BR/BROTHER-36457.mp4'
/var/www/signbank/live/sb-env/lib/python3.5/shutil.py in copyfile, line 114
/usr/bin/uwsgi
3.5.2
['.',  '',  '/var/www/signbank/live/sb-env/lib/python35.zip',  '/var/www/signbank/live/sb-env/lib/python3.5',  '/var/www/signbank/live/sb-env/lib/python3.5/plat-x86_64-linux-gnu',  '/var/www/signbank/live/sb-env/lib/python3.5/lib-dynload',  '/usr/lib/python3.5',  '/usr/lib/python3.5/plat-x86_64-linux-gnu',  '/var/www/signbank/live/sb-env/lib/python3.5/site-packages']
Fri, 21 Jun 2019 07:34:55 +0200

**Comment by Woseseltops** [comment]
Hey @ocrasborn , I'm trying to reproduce this problem, but I'm not sure how exactly you were changing the lemma you created? From which page, which button?

**Comment by ocrasborn** [comment]
Oops... If I remember wellco weren't too menu Signs > show all lemmas, selected it there

**Comment by susanodd** [comment]
When you go to Gloss List View, you have the bar under the List View/ Lemma View/ Number of Hits row where you can add a new gloss.

(see images that step through creation)

I couldn't generate an error either. I created a new lemma during creation of a gloss as follows from the Gloss List view (filled in both language fields), then created annotations (filled in both fields), then did Create and it goes to the Gloss Detail View for the new gloss as shown.
(With the newly created Lemmas and Annotations shown.)

So it must be causing the Video error when it goes to Detail View for the new Gloss/Lemma ?

![gloss-list-create-gloss](https://user-images.githubusercontent.com/24272544/60028088-02a75d00-969f-11e9-8533-2f8aecc87277.png)

![gloss-list-create-gloss-annotation](https://user-images.githubusercontent.com/24272544/60028132-13f06980-969f-11e9-8049-0b832ae66385.png)

![gloss-detail-create-gloss](https://user-images.githubusercontent.com/24272544/60028140-18b51d80-969f-11e9-86a9-ee04d8f44208.png)

**Comment by susanodd** [comment]
Here's another go, using the Add new gloss from the pull-down menu.

![create-gloss-with-new-lemma](https://user-images.githubusercontent.com/24272544/60029048-ae9d7800-96a0-11e9-8a4a-649a566eea2a.png)
![create-gloss-with-new-lemma-detail](https://user-images.githubusercontent.com/24272544/60029059-b3622c00-96a0-11e9-839b-aa563ced3c7d.png)

**Comment by susanodd** [comment]
I didn't try it yet on an empty dataset.

**Comment by vanlummelhuizen** [comment]
What I found/know so far:

The url https://signbank.science.ru.nl/dictionary/lemma/add/36457 suggests @ocrasborn was the Gloss detail view of gloss 36457 when the error occurred. 

I tested the following on my local instance:

1. Create a new dataset in the admin
2. Created a new gloss via /signs/add/, including a new lemma
3. On the gloss detail view page of the new gloss, I created another new lemma

... and I got an error. So, I could reproduce it. Stay tuned!
